e2e/integration/wordpress-01/special-characters-in-url.spec.ts

```diff
@@ -0,0 +1,24 @@
+// This allows us to get TypeScript Intellisense and autocompletion.
+import type { taskTypes } from "../../plugins";
+const task: taskTypes = cy.task;
+
+describe("Special characters in the URL", () => {
+  before(() => {
+    task("loadDatabase", {
+      path: "./wp-data/post-with-special-characters/dump.sql",
+    });
+  });
+
+  it("Should load the post", () => {
+    cy.visit(
+      "http://localhost:3001/test%e2%99%afpost%f0%9f%98%8a/?frontity_name=wp-basic-tests"
```

Comment:
Shouldn't we use here the special characters in the URL? This is the encoded version and it is already in lowercase. :thinking:

Reply:
You're absolutely right, I think pushing this late in the evening was not the best idea.

Just one thing to note, my very first idea was to do just:

```
cy.visit("http://localhost:3001/test♯post😊/?frontity_name=wp-basic-tests");

cy.location("href").should(
      "eq",
      "http://localhost:3001/test%e2%99%afpost%f0%9f%98%8a/"
    );
```

However, the problem is that cypress fails with 
<img width="334" alt="Screen Shot 2020-11-05 at 7 17 10 PM" src="https://user-images.githubusercontent.com/5417266/98310645-82d01900-1f9b-11eb-93d2-73e6be274bd3.png">

The problem is that cypress does not encode the URL for you in the same way that the browser does when you just paste something in the search bar! 😅 More details: https://github.com/cypress-io/cypress/issues/5274#issuecomment-537659134

So, I've created the test case which does the navigation on the client side instead (works fine as expected).